src/main/java/org/mule/extension/http/internal/listener/profiling/tracing/HttpListenerCurrentSpanCustomizer.java

```diff
@@ -15,9 +15,14 @@
 import org.mule.extension.http.api.HttpRequestAttributes;
 import org.mule.extension.http.internal.request.profiling.tracing.HttpCurrentSpanCustomizer;
 import org.mule.sdk.api.runtime.source.DistributedTraceContextManager;
+import org.mule.runtime.api.util.MultiMap;
```

Comment:
We have an order for the imports:

static mule packages
static java, javax packages
static other packages

after that:

mule packages
java, javax packages
rest of the packages.

Reply:
Thank you. I applied it for other classes as well.